src/main/java/core/basesyntax/file/reader/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.file.reader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {
+            List<String> fruitList = new ArrayList<>();
+            reader.readLine();
```

Comment:
```suggestion
```
let's not skip the first line
your `fileReader` should be able to read from any file, csv or non-csv

skip first line later in your parser